source/renderer/app/components/voting/voting-registration-wizard-steps/VotingRegistrationStepsEnterPinCode.tsx

```diff
@@ -162,15 +161,9 @@ class VotingRegistrationStepsEnterPinCode extends Component<Props> {
         activeStep={activeStep}
         actions={actions}
         containerClassName={styles.component}
-        nextFundNumber={nextFundNumber}
       >
         <p className={styles.description}>
-          <FormattedHTMLMessage
-            {...messages.description}
-            values={{
-              nextVotingFundNumber: nextFundNumber,
-            }}
-          />
+          <FormattedHTMLMessage {...messages.description} />
```

Comment:
Can we use just `intl.formatMessage(messages.description)` here?

Reply:
Resolved in c076a3917e896a89cfb497d66c21828eacb5ecd3, thanks!